sources/engine/Stride.VirtualReality/OpenXR/OpenXrTouchController.cs

```diff
@@ -0,0 +1,167 @@
+using System;
```

Comment:
Need copyright header (applies to other files)
(might need to check copyright permission with original author).